community/modules/scripts/pbspro-install/variables.tf

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "rpm_url" {
+  description = "Path to PBS Pro RPM file for select PBS host type (server, client, execution)"
+  type        = string
+}
+
+variable "pbs_exec" {
+  description = "Root path in which to install PBS"
+  type        = string
+  default     = "/opt/pbs"
+}
+
+variable "pbs_data_service_user" {
+  description = "PBS Data Service POSIX user"
+  type        = string
+  default     = "pbsdata"
+}
+
+variable "pbs_home" {
+  description = "PBS working directory"
+  type        = string
+  default     = "/var/spool/pbs"
+}
+
+variable "pbs_license_server" {
+  description = "IP address or DNS name of PBS license server"
+  type        = string
+  default     = "CHANGE_THIS_TO_PBS_PRO_LICENSE_SERVER_HOSTNAME"
```

Comment:
Why not just make this a mandatory variable? I'm assuming this default doesn't provide any value.

Reply:
There is a bit of a challenge here in that this input is required only for the use case of installing the server. So, it can either be required for use cases where it is not actually required or it can be never strictly required. Let me know what you think.

Reply:
That makes sense. I'd suggest adding that to the description, because it's not clear on first glance that this is only sometimes mandatory.

"IP address or DNS nam of PBS license server, mandatory when installing the pbs server."

Reply:
I will address this in a shortly forthcoming PR.